_faq/security_implications/read-data.md

```diff
@@ -0,0 +1,15 @@
+---
+title: How is Forseti able to read data from all my projects?
+order: 2
+---
+{::options auto_ids="false" /}
+
+Forseti uses a service account which is granted roles on the organization 
+IAM policy. Because roles are hierarchical in GCP, any role that someone has 
+on the organization level IAM will also be inherited on a lower level, such as 
+at the folder or project level. For example, if you grant the "Browser" role 
```

Comment:
"... if someone has a Cloud IAM role on the organization level, the role is inherited by lower levels like the folder or project."